ConsoleView.h

```diff
@@ -11,7 +19,32 @@ class ConsoleView
 public:
     ConsoleView(){};
   
-    void UserInputLoop(){};
+    
+    
+    
+    // PrintClass Name will be used to Print the names of the class being passed by the user
+    //@ Para: UMLClass object 
+    void PrintClassName(UMLClass X){}
```

Comment:
- The header file doesn't need the {} at the end.
- The header file also needs a semi colon at the end of every method declaration.